cmd/blocking-issue-creator/main.go

```diff
@@ -136,6 +132,17 @@ func manageIssues(client githubClient, githubLogin string, repoInfo *config.Info
 		return err
 	}
 
+	if len(branches) == 0 {
```

Comment:
I think it would be useful to have a info-level log here about the decision, something like "Repository does not have any blocked branches, deleting blocker issues", maybe conditionally on `len(issues) > 0`